### PR TITLE
port to 2.0

### PR DESCRIPTION
**Pull request by notwaldorf**
Straight forward hybrid port

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -23,16 +23,23 @@
...
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
+    "paper-styles": "PolymerElements/paper-styles#2.0-preview",
     "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "webcomponentsjs": "webcomponents/webcomponentsjs#v1",
+    "test-fixture": "PolymerElements/test-fixture#custom-elements-v1"
   },
-  "main": "app-localize-behavior.html"
+  "main": "app-localize-behavior.html",
+  "resolutions": {
```
Comment (notwaldorf) [inline bower.json]: Ugh, this is needed while `paper-toggle-button` isn't ported and has Opinions™️

File: test/basic.html [hunk 2]
```diff
@@ -130,8 +130,13 @@
...
         // it's just 10$. This isn't something we control, so check the string
         // is mostly right.
-        assert.equal(app.$.output.innerHTML.indexOf('Tout est à 10,00&nbsp;$'), 0);
-        
+
+        // It gets better: on IE11 Win8.1, format.js returns a " " in between
+        // the 10,00 and the $. On every other browser, including IE11 on Win10,
+        // it returns an &nbsp;. Yes, this took an hour to find.
+        var translation = app.$.output.textContent.replace('\u00a0', ' ');
+        assert.equal(translation.indexOf('Tout est à 10,00 $'), 0);
```
Comment (bicknellr) [inline test/basic.html]: Does `assert.equal(translation, 'Tout est à 10,00 $');` work?

Comment (notwaldorf) [inline test/basic.html]: no, because read the comment above -- in non-edge, you get 10 $US (with the currency), but in edge just 10$

Comment (bicknellr) [inline test/basic.html]: Ah, thanks; I only read the new comment.

File: app-localize-behavior.html [hunk 3]
```diff
@@ -215,6 +215,9 @@
     var proto = this.constructor.prototype;
 
     // Everytime any of the parameters change, invalidate the strings cache.
+    if (!proto.__localizationCache) {
+      proto.__localizationCache = {};
```
Comment (bicknellr) [inline app-localize-behavior.html]: This doesn't need to fill all the properties of `__localizationCache` like [this line](https://github.com/PolymerElements/app-localize-behavior/pull/54/files#diff-eb3c5a91b266417d17c72bdbbdf0f0e8R184)?

**Comment by notwaldorf** [comment]
@bicknellr fixed initialization bit, used the right `wc-ce` flag in test, added `iron-component-page` back in

File: bower.json [hunk 4]
```diff
@@ -21,18 +21,36 @@
...
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.6",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.1"
   },
-  "main": "app-localize-behavior.html"
+  "main": "app-localize-behavior.html",
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "iron-ajax": "PolymerElements/iron-ajax#^1.0.0",
+        "polymer": "Polymer/polymer#^1.1.0"
```
Comment (bicknellr) [inline bower.json]: We discussed this offline but we need to change this to `^1.7.0` or maybe `^1.8.0` (here and in all of the other hybrid elements).

Comment (notwaldorf) [inline bower.json]: I think this should be a tedium run -- doing this by hand in all of them is annoying

**Pull request merged**